### ISSUE TITLE
Type presence is not checked

### ISSUE THREAD

**Issue by ganlhi** [whizark/commitlint-config-cz#24]
When using `cz` as config for commitlint, the presence of the type is not checked, only its value if it is present. or instance:

- `my bad commit` : commitlint does not throw an error
- `foo: my bad commit` : commitlint throws an error about the type value

But, if I configure commitlint to extend `@commitlint/config-conventional`, both cases throw errors.

**Comment by whizark** [comment]
@ganlhi Sorry for my late reply.
Could you try to add [type-empty](https://commitlint.js.org/#/reference-rules?id=type-empty) rule to your commitlint config?

`commitlint-config-cz` only adds [scope-enum](https://commitlint.js.org/#/reference-rules?id=scope-enum) and [type-enum](https://commitlint.js.org/#/reference-rules?id=type-enum) rules so empty type value is allowed by default.

Adding `type-enum` rule, commitlint should throw an error with empty type value.

**commitlint.config.js:**

```js
module.exports = {
    extends: [
        'cz'
    ],
    rules: {
        'type-empty': [2, 'never']
    }
};
```

**Comment by whizark** [comment]
I'm closing this issue for now because I cannot reproduce it. Please, feel free to reopen if it's still an issue.
